_posts/2019-12-3-paper-walls.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: article
+title: "Tearing Down the Paper Walls: Valuing Practical Problem-solving in Academia"
+subnav: blog
+comments: true
+tagline: "While at KubeCon last week in San Diego, one "
+author: '<a href="https://ssl.engineering.nyu.edu/personalpages/jcappos/">Justin Cappos</a>'
+categories:
+  - 'Informational'
+---
+
+While at KubeCon last week in San Diego. one of the 12K attendees present came up to me and said, ''You're at NYU. What are you doing at KubeCon?''  While I was unsurprised by the question, I was surprised to learn that the attendee was a PhD student at a major US university.  [Anonymity provided so as not to embarrass anyone.]  The student was even more surprised that an academic like myself had created two CNCF projects and was active in the community.
+
+I met with the student's advisor a bit later and chatted about what brought them to the conference.  The response was that the professor and student were here to learn about how the technology they were researching is used in practice. Their hope was that this might help them write better papers for a major academic conference. However, the professor realized that academia wasn't looking at the same problems. Industry solutions are complex as a necessary part of handling real world problems and thus are both too difficult to package into academic papers, and are harder still to improve on.  So as I understand it, the professor and his student will not be back to the conference because what they heard and saw at this meeting was not sufficiently useful in their endless quest to publish papers.
```

Comment:
```suggestion
I met with the student's advisor a bit later and chatted about what brought them to the conference.  The response was that the professor and student were here to learn about how the technology they were researching is used in practice. Their hope was that this might help them write better papers for a major academic conference. However, the professor realized that academia wasn't looking at the same problems. Industry solutions are complex as a necessary part of handling real world problems and thus are both too difficult to package into academic papers, and are harder still to improve on.  So as I understand it, the professor and student will not be back to the conference because what they heard and saw at this meeting was not sufficiently useful in their endless quest to publish papers.
```